.github/workflows/cd.yml

```
@@ -21,8 +21,8 @@ jobs:
     - name: Get tag and tracker versions
       id: version
       run: |
-        echo ::set-output name=TAG_VERSION::${GITHUB_REF#refs/*/}
-        echo "##[set-output name=PYTHON_TRACKER_VERSION;]$(python setup.py --version)"
+        echo :: "{name}=TAG_VERSION::${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+        echo "##["{name}=PYTHON_TRACKER_VERSION" >> $GITHUB_OUTPUT;]$(python setup.py --version)"
```

I think it should be this if I understand what it's trying to do correctly...
```suggestion
        echo "TAG_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
        echo "PYTHON_TRACKER_VERSION=$(python setup.py --version)" >> $GITHUB_OUTPUT
```

maybe... But I don't really get what that python tracker version line was actually doing before...